Examples/StylesAndOptions/Example/ChooseOptions.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ChooseOptions.swift
+//  Example
+//
+//  Created by Nataliya Patsovska on 2018-06-12.
+//  Copyright © 2018 iZettle. All rights reserved.
+//
+
+import Flow
+import Presentation
+
+struct NamedPresentationOptions {
```

Comment:
This can potentially just be a type alias to (name: String, options: PresentationOptions)

Reply:
I find the struct more convenient since it allows me to use a helper for creating data sources for types conforming to CustomStringConvertible